test/upgrade/upgrade_test.go

```diff
@@ -42,14 +44,16 @@ func TestServerlessUpgrade(t *testing.T) {
 		Tests: pkgupgrade.Tests{
 			PreUpgrade:  preUpgradeTests(),
 			PostUpgrade: postUpgradeTests(ctx),
-			Continual: []pkgupgrade.BackgroundOperation{
-				servingupgrade.ProbeTest(),
-				servingupgrade.AutoscaleSustainingWithTBCTest(),
-				servingupgrade.AutoscaleSustainingTest(),
-				eventingupgrade.ContinualTest(),
-				kafkaupgrade.ChannelContinualTest(),
-				kafkaupgrade.SourceContinualTest(),
-			},
+			Continual: merge(
+				[]pkgupgrade.BackgroundOperation{
+					servingupgrade.ProbeTest(),
+					servingupgrade.AutoscaleSustainingWithTBCTest(),
+					servingupgrade.AutoscaleSustainingTest(),
+					eventingupgrade.ContinualTest(),
+				},
+				kafkaupgrade.ChannelContinualTests(continual.ChannelTestOptions{}),
```

Comment:
This func returns also, `continual.BrokerBackedByChannelTest` which configures config map directly, but it should configure KnativeEventing CR options. Same is true, for knative/operator so this requires some additional upstream enhancements in short term. For now I think I go for local impl. in here, in serverless-operator.

/hold

Reply:
For now, I've just removed the https://github.com/openshift-knative/serverless-operator/pull/983/commits/d02bd041d869137b966f54d8ace1fd7a9cda2d80 commit.

I plan to implement it properly on upstream to enable configuring ConfigMap directly or via CR.

/unhold